src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -501,6 +502,11 @@
 			$ph1ent['mobile'] = true;
 		} else {
 			$ph1ent['remote-gateway'] = $pconfig['remotegw'];
+			if ( !empty($pconfig['ikeport']) && $pconfig['ikeport'] != '500' ) {
```

Comment:
Not exactly here, but up higher in the input validation, this should also test the value of `ikeport` using `is_port()` to ensure the user submitted a sane value.